src/assets/styles/layout.css

```diff
@@ -1,234 +0,0 @@
-article .hero > section,
-article .hero div > section,
-article .main > section,
-article .asides aside > section {
-  max-width: 1060px;
-  margin-right: auto;
-  margin-left: auto;
-  padding-right: 15px;
-  padding-left: 15px;
-}
-@media screen and (min-width: 768px) {
-  article .hero > section,
-  article .hero div > section,
-  article .main > section,
-  article .asides aside > section {
-    padding-right: 30px;
-    padding-left: 30px;
-  }
-}
-::selection {
-  background: #f59f51;
-  color: #fff;
-}
-main {
-  display: block;/* IE 11 gives auto/no height to main */
-  min-height: 200px;
-  background: #f0f0f0;
-  padding-top: 50px;
-  position: relative;
-}
-@media screen and (min-width: 768px) {
-  main {
-    min-height: 400px;
-    padding-top: 73px;
-  }
-}
-h1,
-h2,
-h3 {
-  color: #343434;
-  font-weight: 100;
-}
-a {
-  color: #368aa2;
-  text-decoration: none;
-}
-a:focus, a:hover {
-  color: #2b7187;
-}
-a:active {
-  color: #2b7187;
-}
-a.current:hover {
-  text-decoration: none;
-}
-strong {
-  font-weight: bold;
-}
-.unbreakable {
-  white-space: nowrap;
-}
-.empty-space {
-  min-height: 320px;
-  background: #e6e6e6;
-  position: relative;
-}
-@media multi-column {
-  .empty-space {
-    min-height: 420px;
-  }
-}
-@media screen and (min-width: 768px) {
-  article .content {
```

Comment:
`.content` doesn't exist in publish. Looks like this came from beta. In Publish, we're giving the .main container a bunch of flex properties but it's not inside a `display: flex` container. Looks like it should be in this .content container. https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-5b8e63d7964be0aaeb36dee44f1e1ca3R60